.github/ISSUE_TEMPLATE/test-protocol.md

```diff
@@ -0,0 +1,55 @@
+---
+name: Test protocol
+about: Document a manual testing cycle
+---
+
+# Test details
+
+* Tested ACM Module Version: {e.g. 1.1.5}
+* Github release: {Link to GH release}
+* Test execution date: {2025-07-29}
+* Tester: {@your-Github-user}
+
+# Test protocol
+
+The detailed test flow is explained in our internal documentation.
+
+## Pre-requisites
+
+- [ ] Configure System Landscape and Formations in BTP Cockpit
+- [ ] Access to Back Office and Storefront possible
```

Comment:
```suggestion
- [ ] Access to back-office and storefront possible
```
based on [this doc](https://doc.oroinc.com/user/concept-guides/administration/menus/) this is how it should be spelled